docs/docs/guides/operate/managing-concurrency.md

```diff
@@ -130,43 +522,360 @@ You can use Dagster's rich metadata to use a schedule or a sensor to only start
   title="No more than 1 running job from a schedule"
 />
 
-## Troubleshooting
+## Troubleshooting Common Concurrency Issues
+
+### Configuration Errors
+
+#### Issue: `tag_concurrency_limits` not working in job configuration
+
+**Problem**: Users report that tag concurrency limits defined in job configuration don't work.
+
+**Solution**: Ensure you're using the correct configuration structure. Tag concurrency limits in job definitions should be nested under the executor configuration:
+
+```python
+# ❌ Incorrect - doesn't work
+my_job = define_asset_job(
+    name="my_job",
+    config={
+        "tag_concurrency_limits": [  # Wrong location
+            {"key": "database", "limit": 1}
+        ]
+    }
+)
+
+# ✅ Correct - works properly
+my_job = define_asset_job(
+    name="my_job",
+    config={
+        "execution": {
+            "config": {
+                "multiprocess": {
+                    "tag_concurrency_limits": [  # Correct location
+                        {"key": "database", "limit": 1}
+                    ]
+                }
+            }
+        }
+    }
+)
+```
+
+#### Issue: Executor configuration not taking effect
 
-When limiting concurrency, you might run into some issues until you get the configuration right.
+**Problem**: Configured executor limits are ignored.
 
-### Runs going to STARTED status and skipping QUEUED
```

Comment:
It looks like this entire section was deleted -- if so, that should be brought back.